packages/core/src/checkout-buttons/strategies/braintree/braintree-paypal-credit-button-strategy.ts

```diff
@@ -37,15 +44,30 @@ export default class BraintreePaypalCreditButtonStrategy implements CheckoutButt
             throw new InvalidArgumentError(`Unable to initialize payment because "options.braintreepaypalcredit" argument is not provided.`);
         }
 
-        const state = await this._store.dispatch(this._checkoutActionCreator.loadDefaultCheckout());
-        const paymentMethod = state.paymentMethods.getPaymentMethodOrThrow(methodId);
-        const currency = state.cart.getCartOrThrow()?.currency.code;
+        let state: InternalCheckoutSelectors;
+        let paymentMethod: PaymentMethod<any>;
```

Comment:
It's better to add BraintreePaypal interface (maybe create a new one) for payment method instead of `any` type

Reply:
Can you check the same in braintree paypal strategy too? I think we missed this one in previous PR